packages/pcre/pcre.7.1.6/files/pcre.install

```diff
@@ -0,0 +1,6 @@
+etc: [
+  "setup.ml"
+  "setup.data"
+  "setup.log"
+  "_oasis_remove_.ml"
+]
```

Comment:
@Chris00 Sounds good, I'll just wait until the new release of oasis2opam then.  No hurry.